dev/com.ibm.ws.config/src/com/ibm/ws/config/xml/internal/ConfigRefresher.java

```diff
@@ -82,12 +79,7 @@ public class ConfigRefresher {
         metatypeTracker.open();
 
         changesEndedHook = new ChangesEndedHook();
```

Comment:
We don't need to keep a reference to this hook after it is added.  Could just do:

```java
        CheckpointPhase.getPhase().addMultiThreadedHook(Integer.MIN_VALUE, new ChangesEndedHook());
```